src/include/connection.h

```diff
@@ -155,15 +155,20 @@ struct __wt_heuristic_controls {
      * The controls below deal with the cleanup of obsolete time window information. This process
      * can be configured on two levels:
```

Comment:
I think we may want to change "on two levels". We need to say something that we are two configuration items, one that is shared among the subsystems which is the number of btrees and another one which is unique to each subsystem - the number of pages.

Reply:
Updated the comment according to the suggestion.